_includes/open_graph.html

```diff
@@ -0,0 +1,25 @@
+<meta property="og:url" content="{{ site.url }}{{ page.url }}" />
+<meta property="og:type" content="article" />
+<meta property="og:title" content="{{ page.title | default: site.title }}" />
+{% if page.summary %}
+  {% assign description = page.summary %}
+{% elsif page.questions %}
+  {% assign description = page.questions | join: " " %}
+{% elsif page.keypoints %}
+  {% assign description = page.keypoints | join: " " %}
+{% elsif include.description %}
```

Comment:
How would this description be added? Is it something the lesson authors would add in a page called `_includes/description.html`? Would it be better as an optional yaml item and use `site.description`?

Reply:
This is added at the liquid include step (an example is in `_layouts/workshop.html`):
```liquid
{% include open_graph.html description="Custom description text here" %}
```

I removed the customization in `_config.yml` because I thought it was overcomplicated - it can be restored if we like.